cmd/git-sync/main.go

```diff
@@ -55,6 +55,8 @@ var flRev = flag.String("rev", envString("GIT_SYNC_REV", "HEAD"),
 	"the git revision (tag or hash) to check out")
 var flDepth = flag.Int("depth", envInt("GIT_SYNC_DEPTH", 0),
 	"use a shallow clone with a history truncated to the specified number of commits")
+var flSubmoduleMode = flag.String("submodule-mode", envString("GIT_SYNC_SUBMODULE_MODE", "recursive"),
+	"Submodule clone directive. Options are 'recursive', 'shallow' and 'off'. Default is recursive")
```

Comment:
nits on wording and style.  Something like:

"how to handle submodules - options are 'recursive', 'shallow' and 'off'."

The default should be handled by the flag lib, so no need to reprint here I think